src/awareness.ts

```diff
@@ -76,13 +77,13 @@ export class Awareness extends ObservableV2<Events> implements YJSAwareness {
     const prevStates = this.states;
     const added: number[] = [];
     const removed: number[] = [];
-    const updated: number[] = [];
-    for (const [yjsClientID, client] of prevStates) {
-      const currClient = states.get(yjsClientID);
-      if (currClient === undefined) {
+    const changed: number[] = [];
+    for (const [yjsClientID, state] of prevStates) {
+      const currState = states.get(yjsClientID);
+      if (currState === undefined) {
         removed.push(yjsClientID);
-      } else if (currClient !== client) {
-        updated.push(yjsClientID);
+      } else if (currState !== state || !f.equalityDeep(currState, state)) {
```

Comment:
I think this is wrong.

In Replicache we often get a new object because of cache evictions. I think the test should be:

```js
} else if (currState !== state && !f.equalityDeep(currState, state))) {
```